multiraft/transport.go

```diff
@@ -100,6 +103,18 @@ func NewLocalRPCTransport() Transport {
 	}
 }
 
+// EnableEvents creates the Events channel. All future clients will send an
+// *interceptMessage to that channel and block until the message is
+// acknowledged. EnableEvents should be called before any clients are created,
+// i.e. prior to calls to Connect.
+func (lt *localRPCTransport) EnableEvents() {
```

Comment:
Ok, will go for the thin version without RPC then. Can you explain what bothers you about the current modifications? I thought about creating an extra client but then it just seemed silly to write the same thing twice with a feature that the other one might also want to use at times.